### PR TITLE
Invoke another pipeline to release the website instead of building in this pipeline

### PR DESCRIPTION
**Pull request by israel-hdez**
This is to break the main "kiali-release" pipeline (which source is in https://github.com/kiali/kiali/blob/master/deploy/jenkins-ci/Jenkinsfile) in order to make it thinner.

From now on, the main pipeline will call the `kiali-website-release` when needed, splitting the release process and de-coupling a little this pipeline/repo from the kiali/kiali.io repo.

Related website repo PR: https://github.com/kiali/kiali.io/pull/335
Related kiali/kiali#2895

### CODE REVIEW

**Comment by israel-hdez** [comment]
@cfcosta Asking a review from you, because you did the initial website release work.

File: deploy/jenkins-ci/Jenkinsfile [hunk 1]
```diff
@@ -25,9 +25,9 @@
  *   - UI_GITHUB_URI
  *      defaultValue: git@github.com:kiali/kiali-ui.git
  *      description: SSH Url of the kiali-ui GitHub repository
- *   - SITE_GITHUB_URI
- *      defaultValue: git@github.com:kiali/kiali.io.git
- *      description: SSH Url of the kiali.io GitHub repository
+ *   - SITE_REPO
```
Comment (cfcosta) [inline deploy/jenkins-ci/Jenkinsfile:28]: I like this.

File: deploy/jenkins-ci/Jenkinsfile [hunk 2]
```diff
@@ -195,7 +179,10 @@ node('kiali-build && fedora') {
         returnStdout: true,
         script: "${backendDir}/deploy/jenkins-ci/bin/determine-release-type.sh").trim()
     }
+
+    buildSite = params.SKIP_SITE_RELEASE != "y" && releaseType == "minor"
```
Comment (cfcosta) [inline deploy/jenkins-ci/Jenkinsfile:183]: minor: I would move this up on the website, ideally you should be able to skim there and understand at least the rules for release.

**Review by cfcosta — APPROVED** [review]
Looks ok to me. I added a minor request change but it's minor, you can merge without it if you disagree.

File: deploy/jenkins-ci/Jenkinsfile (hunk 2 above)
Comment (israel-hdez) [reply deploy/jenkins-ci/Jenkinsfile:183]: I'm not sure what you meant for "moving up" :-/
Anyway, this is a partial PR. There is still more work for #2895. So I'll merge and fix in a follow-up when you clarify ;)

**Pull request merged**